Backend/Backend/Controllers/ProfileController.cs

```diff
@@ -153,6 +153,43 @@ public async Task<ActionResult> GetProfileAvatar()
             return BadRequest(e.Message);
         }
     }
+
+    [AllowAnonymous]
+    [HttpPost("sentForgotPasswordEmail")]
+    public async Task<ActionResult> SentForgotPasswordEmail([FromBody] ForgotPasswordEmailRequest request) {
+        try {
+            this.LogDebugControllerAPICall(_logger);
+            
+            if (!ModelState.IsValid) {
+                return BadRequest(ModelState);
+            }
+
+            await _profileService.SentForgotPasswordEmail(request.Email);
+            return Ok($"Email sent to {request.Email}");
+        }
+        catch (Exception e) {
+            this.LogErrorAPICall(_logger, e);
+            return BadRequest(e.Message);
+        }
+    }
+
```

Comment:
Fairly certain you need the AllowAnonymous here as well right?

Reply:
Done